qa/workunits/rbd/cli_migration.sh

```diff
@@ -199,6 +242,61 @@ EOF
     remove_image "${dest_image}"
 }
 
+test_import_native_format_with_namespace() {
+    local base_namespace=$1
+    local base_image=$2
+    local dest_namespace=$3
+    local dest_image=$4
+
+    # Fetch pool_id for RBD pool
+    local pool_id=$(ceph osd pool ls detail --format xml | xmlstarlet sel -t -v "//pools/pool[pool_name='rbd']/pool_id")
+
+    # Function to generate the source spec JSON
+    create_spec() {
+        local namespace=$1
+        local image_name=$2
+        cat > ${TEMPDIR}/spec.json <<EOF
+{
+  "type": "native",
+  "pool_id": ${pool_id},
+  "pool_namespace": "${namespace}",
```

Comment:
> Have you considered adapting test_import_native_format() to work with namespace-containing image specs? 

tried but as it involves with different scenario instead of messing with it created separate function,
but after your suggestion on how to call same function for namespace based images
I thought this would be much easier than earlier with some tweaks to existing,
Thanks for the logic!
now implemented that way.